src/controllers/softtakeover.h

```diff
@@ -13,13 +13,11 @@
 
 class ControlObject;
 
-// 3/128 units away from the current is enough to catch fast non-sequential moves
-//  but not cause an audibly noticeable jump.
-
-
 class SoftTakeover {
   public:
-    static const double kDefaultTakeoverThreshold;
+    // 3/128 units away from the current is enough to catch fast non-sequential moves
+    //  but not cause an audibly noticeable jump.
+    static const double kDefaultTakeoverThreshold = 3.0 / 128;
```

Comment:
Did you test this with non c++11 settings and on all targets? I thought initializing a static variable in the class definition is not allowed.